templates/endpoints_v2/README.md

```diff
@@ -14,17 +14,25 @@ You can also find more advanced tutorials in the `examples/` folder, including t
 
 ## Step 1 - Run the model locally in the Workspace
 
-The llm-serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:
-1. The `models` config in `llm-serve-*.yaml` contains a list of YAML files for the models you want to deploy. You can run any of the models in the `models` directory or define your own model YAML file and run that instead. All config files follow the naming convention `{model_name}_{accelerator_type}_{tensor_parallelism}`. Follow the CustomModels [guide](examples/CustomModels.ipynb) for bringing your own models.
-2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN variable to be set to a Hugging Face Access Token for an account with permissions to download the model.
+We provide a starter command to allow you to run Llama-2 and Mistral-family models via Ray Serve. You can specify the model ID, GPU type, tensor parallelism via the command arguments. You can also follow the [guide](examples/CustomModels.ipynb) for bringing your own models.
 
+Currently tensor parallelism defaults to 1 if not specified.
 
+Please note that the Meta Llama-2 family of models need the `hf_token` variable to be set to a Hugging Face Access Token for an account with permissions to download the model. You can get your token [here](https://huggingface.co/settings/tokens).
 
-From the VSCode terminal (press [**Ctrl + `**] in VSCode), use the Ray Serve CLI to deploy the model for testing. It will take a few minutes to initialize and download the model.
 
-```bash
-# Note: if using GCP cloud, use llm-serve-gcp.yaml instead to select L4 GPU instances.
-$ serve run llm-serve-aws.yaml
+
+```python
+# Example command to serve Mistal-7B via A10 GPUs on AWS
+!serve run rayllm.start:endpoint model_id=mistralai/Mistral-7B-Instruct-v0.1 gpu_type=A10
```

Comment:
Do we have any entrypoint that dumps out the config?

Reply:
Talked to Shryeas and `serve run` doesn't have a good way to outputting the config rn.

Filed an issue to track https://anyscale1.atlassian.net/browse/AE-634